pkg/cluster_upgrader/cluster_upgrader_test.go

```diff
@@ -3,6 +3,7 @@ package cluster_upgrader
 import (
 	"context"
 	"fmt"
+	"github.com/openshift/managed-upgrade-operator/pkg/scaler"
```

Comment:
put this in bottom line ? we should better group the imports

Reply:
I just rely on `go fmt` for ordering. It keeps it consistent. What order do you suggest?

Reply:
Split the imports like this with an empty line : https://github.com/openshift/cluster-version-operator/blob/master/pkg/cvo/availableupdates.go#L4-L18 ?

Reply:
Can do. `go fmt` seems to respect the groupings